indigo_content_api/v2/views.py

```diff
@@ -370,8 +386,12 @@ def get_publication_document(self, request, filename, *args, **kwargs):
         raise Http404()
 
 
+@extend_schema(responses=TaxonomyTopicSerializer)
 class TaxonomyTopicView(ContentAPIBase, viewsets.ReadOnlyModelViewSet):
-    queryset = TaxonomyTopic.get_public_root_nodes()
+    """API endpoint for listing taxonomy topics. Taxonomy topics are a tree structure of topics that are applied to
+    documents to categorise them.
+    """
+    queryset = TaxonomyTopic.objects.none()
```

Comment:
are we still calling `get_public_root_nodes()` elsewhere that I'm not seeing? or how does this work now?

Reply:
yes, this is just so that drf-spectacular knows what the model is; the actual queryset is determined in get_queryset

i've added a comment to show this